modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,425 @@
+
+- `consumer_project_id`: The ID of the GCP project where consumer (client) resources will be created. Should match one of the projects included in the `consumer_accept_list` field in previous steps.
+- `consumer_vpc_id`: The ID of the VPC where the consumers will be deployed.
+- `dns_a_records`: The list of DNS A records for each Redpanda broker API. These will be provided by the Redpanda team and will never change. Format: `["<record 1>", "<record 2>", "<record N>"]`. *All records must end with a dot*.
+- `dns_zone`: The base DNS zone for the records. Can be derived from them. *The DNS zone name must end with a dot*.
+- `dns_a_records_target_ip`: The IP address that the A records will point to, which should be that of the forwarding rule deployed in the previous step. You can query the target IP address with the following `gcloud` command:
++
+[,bash]
+----
+gcloud compute forwarding-rules describe redpanda-psc-endpoint --region <region> --project <consumer project ID>
+----
+
+[,bash]
+----
+cd terraform/3_dns
+cat << EOF > input.tfvars
+consumer_project_id = "<consumer GCP project ID>"
+consumer_vpc_id = "<consumer VPC ID>"
+dns_a_records = <list of broker A records>
+dns_zone = "<base Redpanda DNS zone>"
+dns_a_records_target_ip = "<PSC consumer forwarding rule IP>"
+EOF
+terraform init
+terraform apply -var-file=input.tfvars 
+----
+
+=== Update your firewall rules
+
+Make sure that your consumer VPC firewall allows traffic to and from the PSC forwarding rule IP address on the expected ports.
+
+== Test the connection
+
+You can test the PSC connection from any VM or container in the consumer VPC. If configuring a client isn't possible right away, you can do these checks using `rpk` or curl:
+
+=== Test Kafka API
+
+. Set the following environment variables.
++
+[,bash]
+----
+REDPANDA_BROKERS='<broker URL>:30292'
+REDPANDA_SASL_MECHANISM='<SCRAM-SHA-256 or SCRAM-SHA-512>'
+REDPANDA_SASL_USERNAME='<username>'
+REDPANDA_SASL_PASSWORD='<password>'
+----
+
+. Check the cluster status using `rpk`.
++
+[,bash]
+----
+rpk cluster info
+----
+
+. Create a test topic.
++
+[,bash]
+----
+rpk topic create psc-test
+----
+
+. Produce to the test topic.
++
+[,bash]
+----
+echo 'hello PSC!' | rpk topic produce psc-test
+----
+
+. Consume from the test topic.
++
+[,bash]
+----
+rpk topic consume psc-test -n 1
+----
+
+== Access Redpanda services through VPC endpoint
+
+The service attachment exposes 2 sets of ports:
+
+* *Seed ports*: These ports load balance requests among brokers. Useful for initiating a connection from a client.
++
+|=== 
+| Redpanda service | Default seed port 
+
+| Kafka API | 30292 
+| HTTP Proxy | 30282 
+| Schema Registry | 30081 
+|===
+
+* *Broker-API-specific ports*: These ports route requests to a specific API on a specific broker. Can be determined from the broker ID or number.
+
+For Kafka API:
+
+- 32092: Kafka API on broker 0
+- 32093: Kafka API on broker 1
+- 32094: Kafka API on broker 2  
+Etc.
```

Comment:
Could we explain what "etc" means here?

Reply:
Clusters might have more than 3 nodes, in which case the list would go on.

Reply:
@0x5d This is the approach we would have taken in the PL docs if we had kept these sections: https://github.com/redpanda-data/cloudv2/blob/main/tools/aws-private-link/README-customer.md#access-kafka-api-service-at-redpanda-broker-node Would something like that work? Also, want to double check that it's `35084` for HTTP Proxy API on broker 2

Reply:
Let's remove this part altogether - customers shouldn't have to worry about this, and they should not use these ports anyway, only seed ports.
Sorry it took me a while to realize it.